.devcontainer/Dockerfile

```
@@ -0,0 +1,38 @@
+# Update the NODE_VERSION arg in docker-compose.yml to pick a Node version: 10, 12, 14
+ARG NODE_VERSION=14
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${NODE_VERSION}
+
+# VARIANT can be either 'hugo' for the standard version or 'hugo_extended' for the extended version.
+ARG VARIANT=hugo
```

Our theme requires the extended version.

See: https://www.docsy.dev/docs/getting-started/

```suggestion
ARG VARIANT=hugo_extended
```